include/rocksdb/statistics.h

```diff
@@ -541,6 +541,8 @@ class Statistics {
 };
 
 // Create a concrete DBStatistics object
+template <uint32_t TICKER_MAX = TICKER_ENUM_MAX,
```

Comment:
Can we keep the existing API unchanged and add the template version of `CreateDBStatistics` to another header file under include/rocksdb/utilities?